redash/tasks/queries.py

```diff
@@ -130,14 +130,15 @@ def enqueue_query(query, data_source, user_id, scheduled_query=None, metadata={}
                     scheduled_query_id = None
                     time_limit = settings.ADHOC_QUERY_TIME_LIMIT
 
-                args = (query, data_source.id, metadata, user_id, scheduled_query_id)
+                args = (query, data_source.id, metadata, user_id, is_api_key, scheduled_query_id)
                 argsrepr = json_dumps({
                     'org_id': data_source.org_id,
                     'data_source_id': data_source.id,
                     'enqueue_time': time.time(),
                     'scheduled': scheduled_query_id is not None,
                     'query_id': metadata.get('Query ID'),
-                    'user_id': user_id
+                    'user_id': user_id,
+                    'is_api_key': is_api_key
```

Comment:
We don't need this here, as we didn't update the monitoring code to use it (and probably won't need to).

Reply:
5770888